plugin/v3/plugin.proto

```diff
@@ -12,6 +12,10 @@ service Plugin {
   rpc GetName(GetName.Request) returns (GetName.Response);
   // Get the current version of the plugin
   rpc GetVersion(GetVersion.Request) returns (GetVersion.Response);
+  // Get plugin own (nested) spec schema.
```

Comment:
```suggestion
  // Get plugin's own (nested) spec schema.
```

Reply:
I cast thy `attributive noun` meaning for `plugin spec` used here!

Reply:
9e8d07bb634c781c024e7a461f3d963919fbadfc